server/src/test/java/org/elasticsearch/index/query/SpanMultiTermQueryBuilderTests.java

```diff
@@ -162,4 +161,95 @@ public void testFromJson() throws IOException {
         assertEquals(json, "ki", ((PrefixQueryBuilder) parsed.innerQuery()).value());
         assertEquals(json, 1.08, parsed.innerQuery().boost(), 0.0001);
     }
+
+    public void testDefaultMaxRewriteWithXContentValue() throws Exception {
+        String json = "{" +
+            "\"span_multi\" : {" +
+            "\"match\" : {" +
+            "\"prefix\" : {" +
+            "\"user\" : {" +
+            "\"value\" : \"ki\"," +
+            "\"boost\" : 1.08" +
+            "}"+
+            "}"+
+            "}," +
+            "\"boost\" :1.0" +
+            "}" +
+            "}";
```

Comment:
The xcontent parsing is tested in the parent class, can you modify this test to use the query builders instead ?